projects/poetry_polylith_plugin/pyproject.toml

```diff
@@ -9,31 +9,32 @@ license = "MIT"
-    {include = "polylith/configuration",from = "../../components"},
-    {include = "polylith/deps",from = "../../components"},
-    {include = "polylith/development",from = "../../components"},
-    {include = "polylith/diff",from = "../../components"},
-    {include = "polylith/dirs",from = "../../components"},
-    {include = "polylith/distributions",from = "../../components"},
-    {include = "polylith/files",from = "../../components"},
-    {include = "polylith/imports",from = "../../components"},
-    {include = "polylith/info",from = "../../components"},
-    {include = "polylith/interface",from = "../../components"},
-    {include = "polylith/libs",from = "../../components"},
-    {include = "polylith/poetry",from = "../../components"},
-    {include = "polylith/project",from = "../../components"},
-    {include = "polylith/readme",from = "../../components"},
-    {include = "polylith/repo",from = "../../components"},
-    {include = "polylith/reporting",from = "../../components"},
-    {include = "polylith/sync",from = "../../components"},
-    {include = "polylith/test",from = "../../components"},
-    {include = "polylith/toml",from = "../../components"},
-    {include = "polylith/workspace",from = "../../components"},
+    { include = "polylith/poetry_plugin", from = "../../bases" },
+    { include = "polylith/alias", from = "../../components" },
+    { include = "polylith/bricks", from = "../../components" },
+    { include = "polylith/check", from = "../../components" },
+    { include = "polylith/commands", from = "../../components" },
+    { include = "polylith/configuration", from = "../../components" },
+    { include = "polylith/deps", from = "../../components" },
+    { include = "polylith/development", from = "../../components" },
+    { include = "polylith/diff", from = "../../components" },
+    { include = "polylith/dirs", from = "../../components" },
+    { include = "polylith/distributions", from = "../../components" },
+    { include = "polylith/files", from = "../../components" },
+    { include = "polylith/imports", from = "../../components" },
+    { include = "polylith/info", from = "../../components" },
+    { include = "polylith/interface", from = "../../components" },
+    { include = "polylith/libs", from = "../../components" },
+    { include = "polylith/poetry", from = "../../components" },
+    { include = "polylith/project", from = "../../components" },
+    { include = "polylith/readme", from = "../../components" },
+    { include = "polylith/repo", from = "../../components" },
+    { include = "polylith/reporting", from = "../../components" },
+    { include = "polylith/sync", from = "../../components" },
+    { include = "polylith/test", from = "../../components" },
+    { include = "polylith/toml", from = "../../components" },
+    { include = "polylith/workspace", from = "../../components" },
+    { include = "polylith/yaml", from = "../../components" },
```

Comment:
I don't think the Poetry plugin need the new component, right?

Reply:
Or maybe it does because of the libs component, sorry! I will review your PR more properly further on!

Reply:
Well, not by itself. But the `polylith/libs` component now imports the `load_yaml` function from the `polylith/yaml` component